[helix] remove default subtitle provider setting

disrupted opened the pull request:
not needed anymore since Helix

disrupted committed:
[change] rename skin to prevent wrong auto-updates

disrupted committed:
[added] xperience1080+ sounds

disrupted committed:
re-add missing header

disrupted committed:
[helix] remove default subtitle provider setting

not needed anymore since Helix

vonH commented:
You're welcome to create your own version, but there's little point submitting merges as this branch is not as updated as the master - which is not on git. Sounds won't be added, missing headers have been fixed and the subtitle provider setting is being built upon.

disrupted commented:
oops, I only submitted one commit to the Pull Request. pushing the other commits to my fork must have messed it up a little. sorry about that.
I only wanted to remove the deprecated subtitle setting. I'm sure you would have done the same at some point, just wanted to help out a little bit, because I removed it anyways.
sorry for the trouble. best regards

disrupted committed:
changes to Landscape & Banner views

Landscape: remove description for TV Shows
Banner: readded the old Banner view for Episodes from Xperience1080+

disrupted committed:
[fix] remove white line

Banner view episode thumbnail fix for 720p TV

disrupted committed:
use xperience1080 textures

disrupted committed:
name change - 1080X

remove the "F" from "1080XF", because I don't use the flat textures
anymore

disrupted committed:
add trakt.tv rating dialog

disrupted committed:
show watched overlay

- remove unwatched overlays, what matters to me are the watched overlays
- fixed some TV Show posters in Episode Info Dialog. previously it was
using the fallback when it couldn't find the season poster

disrupted committed:
remove flat overlays

use xperience1080 overlays

disrupted committed:
use ClearLogo of TV Show in Now Playing view

disrupted committed:
update navigation headers

- remove pagecounter
- fix Recently Added Episodes & Movies

disrupted committed:
add "Show Plot" option for Landscape views

from xperience1080+

disrupted committed:
syncing upstream changes

bump to 3.40.51

disrupted committed:
add missing header

vonH committed:
[sound] adds singular sound for actions

vonH committed:
[fix] final fix of weather and time layout

vonH committed:
[helix] removal of non-required option

vonH committed:
[texture] new clock texture

vonH committed:
[texture] new clock texture

vonH committed:
[version] 3.40.55

vonH committed:
[version] 3.40.55

vonH committed:
[fix] adds codec information to music

vonH committed:
[version] 3.40.55

vonH committed:
[icon] refreshed logo

disrupted committed:
add trakt.tv options to context menu

disrupted committed:
merge upstream changes

vonH committed:
[fix] removes debugging string

vonH committed:
[version] 3.40.56

vonH committed:
[volume] changes from db to %

vonH committed:
[volume] changes texture size

vonH committed:
[version] 3.40.57

vonH committed:
[fix] youtube trailer url correction

vonH committed:
[version] 3.40.58

disrupted committed:
revert skin debugging

disrupted committed:
[fix] Banner episode view watched indicators

disrupted committed:
[global search] remove redundant result indicator at the top left corner

disrupted committed:
add watched indicators to InfoList season view

disrupted committed:
merge upstream changes - version 3.40.58

disrupted committed:
[fix] OSD clock animation

freeman212 committed:
[fix] slightly smaller ClearLogo on seekbar

freeman212 committed:
[fix] TV Show poster

show poster of TV Show if ClearLogo isn't found

freeman212 committed:
Header: show Video Addon name

Don't show "Video Add-ons: <Add-on name>", but only  Add-on name itself

freeman212 committed:
Video Info: "Poster" instead of "Thumbnail"

freeman212 committed:
[cosmetic] capitalisation "TV Guide"

freeman212 committed:
[cosmetic] capitalisation "TV Shows"

freeman212 committed:
[cosmetic] further homescreen capitalisation

vonH committed:
[texture] correction

vonH committed:
[texture] new channel button

vonH committed:
[texture] new EPG button

vonH committed:
[texture] new text-livetv button

vonH committed:
[texture] new LiveTV defaults

vonH committed:
[texture] correction epg-timer icon

vonH committed:
[texture] new OSD record buttons

vonH committed:
[texture] new textures for FanartSlide view

vonH committed:
[cosmet] fixes volume alignment

vonH committed:
[additional] adds ReloadSkin option to SkinSettings

vonH committed:
[additional] adds ReloadSkin option to SkinSettings

Incs. req.

vonH committed:
[additional] new Fanart view for videotypes

vonH committed:
[version] 3.40.70

vonH committed:
[version] 3.40.70

vonH committed:
[version] 3.40.71

disrupted committed:
merge upstream changes - version 3.40.71

vonH committed:
[texture] updates for xperience1080

vonH committed:
[texture] updates for xperience1080

vonH committed:
[texture] updates for default

vonH committed:
[font] additional

vonH committed:
[viewtype] changes to Fanart Viewtype

vonH committed:
[viewtype] changes to Fanart Viewtype

vonH committed:
[settings] further Skin Setting options

vonH committed:
[home] extensive changes to home screen

Creates a new unified home screen whilst retaining functionality and
flow

vonH committed:
[version] 3.44.00

vonH committed:
[icon] updated

vonH committed:
[icon] updated

disrupted committed:
move texture

vonH committed:
[resolved] fixes list view movie sets icon display

vonH committed:
[resolved] fixes YouTube activ. code not disp.

Fixes the issue with the YouTube add-on activation code not displaying.

vonH committed:
[New] changes section tiles to match parent

vonH committed:
[New] re-order of menu items - Exit is first

vonH committed:
[version] 3.44.06

vonH committed:
[epg] changes to EPG layout

vonH committed:
[version] 3.40.08

freeman212 committed:
merge upstream changes - version 3.44.06

keeping original homescreen design

freeman212 committed:
[fix] addon,xml merge conflicts

freeman212 committed:
revert homescreen layout changes

freeman212 committed:
remove textures

freeman212 committed:
[now playing] show plot for streams

especially youtube

freeman212 committed:
merge upstream changes - version 3.44.08

vonH committed:
[fix] fixes offset pixel on slide selection

Previous version was off by 2 pixels on the left, 3 on the right. Fixes
for Spotlight, Movies slide widget, TV slide widget and Music.

vonH committed:
[fix] fixes CPU usage for linux systems

vonH committed:
[version] 3.40.10

vonH committed:
[texture] pause texture

vonH committed:
[texture] pause texture

vonH committed:
[new] adds pause to player.paused

vonH committed:
[version] 3.44.12

vonH committed:
[tint] fixes tile tinting

vonH committed:
[moving pause] fixes moving pause image

vonH committed:
[version] 3.44.14

freeman212 committed:
merge upstream - version 3.44.14

skipping most changes

freeman212 committed:
remove new pause textures

don't need them

disrupted committed:
[home] xperience1080+ tile colors tiles

Music tiles red + Weather tiles orange

disrupted committed:
[home] remove shadow from menu labels

- remove shadow from menu labels, shutdown and favourites menu

disrupted committed:
[home] fix OpenELEC settings shortcut

pressing up from the settings menu takes you to the OpenELEC settings
addon

disrupted committed:
move fade backgrounds to media folder

will be placed inside the Textures.xbt file

disrupted committed:
remove old screenshots

disrupted committed:
remove redundant visibility condition

disrupted committed:
[fix] TVGuide weekday labels: number of episodes

didn't always show the number episodes on TV for each weekday
